app/src/main/AndroidManifest.xml

```diff
@@ -132,7 +132,7 @@
         <activity
             android:name=".activity.DocumentActivity"
             android:label="@string/app_name"
-            android:launchMode="standard"
+            android:launchMode="singleTop"
```

Comment:
This destorys the "Multiple Windows" feature. This way you can only have one file open a time.

Reply:
Ah. Will fix

Reply:
Actually, could you confirm that making `DocumentActivity` `singleTop` actually prevents multiple windows. I am able to open multiple windows just fine with singleTop enabled.